### ISSUE TITLE
Table border is not showing in Firefox browser

### ISSUE THREAD

**Issue by dbelob** [primefaces/primeng#11010]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
https://stackblitz.com/edit/primeng-tablecolresize-demo
https://www.primefaces.org/primeng/showcase/#/table/colresize
(open in _Firefox_)

**Current behavior**
Table border is not visible in _Firefox_:
![image](https://user-images.githubusercontent.com/22131253/147577455-1155b271-0bc4-4fc4-b12f-f0facb3c5fdd.png)

**Expected behavior**
Table border should be visible in any browser:
![image](https://user-images.githubusercontent.com/22131253/147577598-d3225c11-bccf-44ec-a751-5c0a7e276682.png)

**Minimal reproduction of the problem with instructions**
1. Add `[resizableColumns]="true"` to `p-table` tag
2. Open webpage in _Firefox_

**What is the motivation / use case for changing the behavior?**
Fix bug.

**Please tell us about your environment:**

* **Angular version:** 13.1.1

* **PrimeNG version:** 13.0.3 (since 10.x.x)

* **Browser:** Firefox 95.0.2
 
* **Language:** all

**Comment by dbelob** [comment]
There is a similar closed issue #5829

**Comment by yigitfindikli** [comment]
Seems to be fixed with https://github.com/primefaces/primeng/issues/11021
